languages/scala/reference/exercise-concepts/hello-world.md

```diff
@@ -0,0 +1,13 @@
+# Hello World
+
+[Example implementation](https://github.com/exercism/scala/blob/master/exercises/hello-world/example.scala)
+
+# General
+
+- learn how exercism works: submit solution
```

Comment:
I don't think this is a Scala Concept ;)

Reply:
I agree, but without this critical pre-requirement you can't do any of the other exercises. 
Shouldn't we keep it ?

Reply:
No I don't think we do, as the new Concept Exercises will unlock purely on Concepts, not through some ordering.

Reply:
ok